client/src/pages/Lobby.js

```diff
@@ -1,24 +1,132 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState, useEffect, useContext } from 'react';
+import { Redirect, useHistory } from 'react-router';
 import socketIOClient from 'socket.io-client';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import Slide from '@material-ui/core/Slide';
+import { ContinueButton } from '../components/Buttons';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import Avatar from '@material-ui/core/Avatar';
+import Grid from '@material-ui/core/Grid';
+import avatar from '../images/avatar.png';
+import { useStyles } from '../themes/theme';
+import { TextField } from '@material-ui/core';
+import { store } from '../context/store';
+import { CopyToClipboard } from 'react-copy-to-clipboard';
 
-const ENDPOINT = '/';
+const Transition = React.forwardRef(function Transition(props, ref) {
+  return <Slide direction='up' ref={ref} {...props} />;
+});
 
-function Lobby() {
-  const [response, setResponse] = useState('');
+const Lobby = () => {
+  const classes = useStyles();
+  const ENDPOINT = '/';
+  const [open, setOpen] = React.useState(true);
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+  const [copied, setCopied] = useState(false);
+  const [userData, setUserData] = useState('');
```

Comment:
Should we be getting userData from the useContext? If not, where/when does this get set?